<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cetak Struk Detail</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string>QWidget#centralwidget { background-color: #E3F2FD; }</string>
   </property>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>QPushButton {
	background-color: #757575;
	color: white;
	border: none;
	border-radius: 5px;
	padding: 5px 10px;
	font: 600 9pt &quot;Times New Roman&quot;;
}
QPushButton:hover {
	background-color: #616161;
}</string>
    </property>
    <property name="text">
     <string>‚Üê Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>411</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CETAK STRUK DETAIL üßæ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_struk">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>500</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>QGroupBox { 
       background-color: white; 
       border: 2px solid #BBDEFB; 
       border-radius: 10px; 
       margin-top: 15px; 
       padding-top: 15px;
       font: 700 12pt &quot;Times New Roman&quot;; 
       color: #1976D2; 
      }
      QGroupBox::title { 
       subcontrol-origin: margin; 
       subcontrol-position: top center; 
       padding: 0 10px; 
      }</string>
    </property>
    <property name="title">
     <string>Preview Struk</string>
    </property>
    <widget class="QLabel" name="label_toko">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>400</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>color: #0D47A1;</string>
     </property>
     <property name="text">
      <string>TOKO PAKAIAN CEPAT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_alamat">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>75</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Jl. Contoh No. 123, Kediri</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_no_struk">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>105</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>No: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tanggal">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tanggal: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_kasir">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>155</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kasir: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="line">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>185</y>
       <width>400</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>background-color: #000000;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_detail">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>400</width>
       <height>250</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>QTextEdit {
	border: 1px solid #BBDEFB;
	border-radius: 5px;
	padding: 5px;
	background-color: white;
	font-family: &quot;Courier New&quot;;
	font-size: 10pt;
}</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier New'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>Detail barang akan muncul di sini...</string>
     </property>
    </widget>
    <widget class="QFrame" name="line_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>460</y>
       <width>400</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>background-color: #000000;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_subtotal">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>475</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sub Total:      Rp 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_diskon">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>500</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Diskon (0%):    Rp 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pajak">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>525</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pajak (0%):     Rp 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_total">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>555</y>
       <width>400</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>color: #D32F2F;</string>
     </property>
     <property name="text">
      <string>TOTAL:         Rp 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_terimakasih">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>590</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Terima kasih atas kunjungan Anda!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_print">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>710</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>QPushButton {
	background-color: #2196F3;
	color: white;
	border: none;
	border-radius: 8px;
	padding: 10px;
	font: 700 12pt &quot;Times New Roman&quot;;
}
QPushButton:hover {
	background-color: #1976D2;
}</string>
    </property>
    <property name="text">
     <string>Print Struk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_kembali">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>710</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>QPushButton {
	background-color: #FF9800;
	color: white;
	border: none;
	border-radius: 8px;
	padding: 10px;
	font: 700 12pt &quot;Times New Roman&quot;;
}
QPushButton:hover {
	background-color: #F57C00;
}</string>
    </property>
    <property name="text">
     <string>Kembali</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
